configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionRepository.java

```diff
@@ -685,6 +687,24 @@ private void copyApp(File sourceDir, File destinationDir) throws IOException {
         }
     }
 
+    // TODO: Remove in Vespa 8 (when we don't allow files in SEARCH_DEFINITIONS_DIR)
+    // Copies schemas from searchdefinitions/ to schemas/ if searchdefinitions/ exists
+    private void copyToSchemasDir(java.nio.file.Path applicationDir) throws IOException {
```

Comment:
Is this code run in the prepare stage, or in activate ?
As long as it does not change set of files between the phases is good.

Reply:
It's done in the prepare stage for an external deploy, so after that nothing should change that deployment. I guess it could happen while upgrading, and for old applications that are distributed between config servers... Will look into it

Reply:
Are we giving a warning when people are using searchdefinitions ?

Reply:
Yes, we are giving a warning